# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsLayoutUtils.h
# Commit: 5005fe56b7b0
# Full Hash: 5005fe56b7b0153d2069d25c6f1f4e311e2c1eb1
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 05:56:31
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 985dccd11e97 -r 5005fe56b7b0 layout/base/nsLayoutUtils.h
--- a/layout/base/nsLayoutUtils.h	Thu Feb 20 21:12:05 2020 +0000
+++ b/layout/base/nsLayoutUtils.h	Thu Feb 20 20:52:10 2020 +0000
@@ -2125,9 +2125,8 @@
     SFE_WANT_FIRST_FRAME_IF_IMAGE = 1 << 1,
     /* Whether we should skip colorspace/gamma conversion */
     SFE_NO_COLORSPACE_CONVERSION = 1 << 2,
-    /* Specifies that the caller wants either OPAQUE or NON_PREMULT mAlphaType,
-       if this is can be done efficiently. */
-    SFE_PREFER_NO_PREMULTIPLY_ALPHA = 1 << 3,
+    /* Caller handles SFER::mAlphaType = NonPremult */
+    SFE_ALLOW_NON_PREMULT = 1 << 3,
     /* Whether we should skip getting a surface for vector images and
        return a DirectDrawInfo containing an imgIContainer instead. */
     SFE_NO_RASTERIZING_VECTORS = 1 << 4,
